ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/tls/TlsConfiguration.scala

```diff
@@ -40,6 +43,7 @@ final case class TlsConfiguration(
           .forClient()
           .keyManager(keyCertChainFile.orNull, keyFile.orNull)
           .trustManager(trustCertCollectionFile.orNull)
+          .protocols(if (protocols.nonEmpty) protocols.asJava else null)
```

Comment:
I assume `protocols(null)` is equivalent to not calling `protocols` and not equivalent to an empty list?

Reply:
Unfortunately that's the case. Passing an empty collection will cause all hell to break loose when using this configuration. Using `null` means that you will use the default.